website/docusaurus.config.js

```diff
@@ -270,7 +268,6 @@ var siteSettings = {
   ],
   plugins: [
     [path.resolve("plugins/insertMetaTags"), { metatags }],
-    path.resolve("plugins/svg"),
```

Comment:
This custom svg plugin was created during the initial Docusaurus setup. It didn't feel necessary, and svgs still appear to work after removing this plugin.